examples/complete/main.tf

```
@@ -1,7 +1,7 @@
-module "example" {
-  source = "../.."
-
-  example = var.example
+module "example_service_quotas" {
+  source = "../../"
 
   context = module.this.context
+
+  service_quotas = var.service_quotas
```

The context line should be last

```suggestion
  service_quotas = var.service_quotas

  context = module.this.context
```